### PR TITLE
Added debug commands for changing map style.

### PR DESCRIPTION
**Pull request by darina**

### CODE REVIEW

File: map/framework.cpp [hunk 1]
```diff
@@ -1394,6 +1394,9 @@ string Framework::GetCountryName(m2::PointD const & pt) const
 
 bool Framework::Search(search::SearchParams const & params)
 {
+  if (ParseDrapeDebugCommand(params.m_query))
```
Comment (ygorshenin) [inline map/framework.cpp]: Could you please group this check with ParseEditorDebugCommand()? For example, you may create a helper function, say:
```c++
// Returns true when |command| is a debug command.
bool ParseDebugCommand(std::string const & command)
...
```
and call it there.

File: map/framework.cpp [hunk 2]
```diff
@@ -2868,6 +2871,29 @@ uint32_t GetBestType(FeatureType const & ft)
...
+  else if (query == "?light" || query == "mapstyle:light")
+    desiredStyle = MapStyleClear;
+  else if (query == "?vdark" || query == "mapstyle:vehicle_dark")
+    desiredStyle = MapStyleVehicleDark;
+  else if (query == "?vlight" || query == "mapstyle:vehicle_light")
+    desiredStyle = MapStyleVehicleClear;
+  else
+    return false;
+
+#ifdef OMIM_OS_ANDROID
```
Comment (ygorshenin) [inline map/framework.cpp]:
```c++
#if defined(OMIM_OS_ANDROID)
```
is the same, but more flexible as you may combine it later with other checks.

File: map/framework.hpp [hunk 3]
```diff
@@ -620,6 +620,7 @@ class Framework : public search::ViewportSearchCallback::Delegate,
   //void GetLocality(m2::PointD const & pt, search::AddressInfo & info) const;
   /// @returns true if command was handled by editor.
   bool ParseEditorDebugCommand(search::SearchParams const & params);
+  bool ParseDrapeDebugCommand(std::string const & query);
```
Comment (ygorshenin) [inline map/framework.hpp]: Comment to the method, please.

File: map/framework.cpp (hunk 2 above)
Comment (darina) [inline map/framework.cpp]: done

File: map/framework.hpp (hunk 3 above)
Comment (darina) [inline map/framework.hpp]: done

**Pull request merged**